### DISCUSSION TITLE
Welcome to Detection Discussions!

### DISCUSSION THREAD

**Discussion by wangkanai** [wangkanai/wangkanai#473] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vankampenp** [comment]
It would be good if Chrome OS can be added to the detected Platforms

Mozilla/5.0 (X11; CrOS x86_64 14092.77.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.107 Safari/537.36

**Reply by wangkanai** [reply]
Thank you, i will add it to the todo list.
